lib/client/config.ts

```diff
@@ -29,4 +29,5 @@ export type Config = BackendAPI &
   BrokerClient &
   BrokerServer &
   CommitSigning &
-  HighAvailabilityMode;
+  HighAvailabilityMode &
+  Record<string, any>;
```

Comment:
@aarlaud, is this "relaxing" of the config we talked about?

Reply:
yes, for now we have to type this generically. I don't think that generic part will go away anytime soon tbh, but we'll add other types that we know about while leaving other values possible. Who knows what new thing we'll need to pass from Node....